requirements.txt

```diff
@@ -3,6 +3,7 @@ click>=6.0,<=7.99
 jsonref>=0.1,<=0.2.99
 requests>=2.12.0,==2.*
 jsonschema>=2.5.1,<=3.99
-joblib>=0.11,<0.15
+pubnub>=4.1.12,<=4.99
+joblib>=0.11,<=1.1.*
```

Comment:
Instead of `<=1.1.*`, what do you think about `<2`? If joblib follows semantic versioning (it does, correct?), then anything below version 2 should be fine, and `<2` would mean we don't have to make a PR here every time joblib releases a new `1.*`.

Reply:
That works for me. I would be okay with that.